sklearn/linear_model/ridge.py

```diff
@@ -327,7 +327,7 @@ def ridge_regression(X, y, alpha, sample_weight=None, solver='auto',
         y = check_array(y, dtype=np.float64, ensure_2d=False, order='F')
     else:
         X = check_array(X, accept_sparse=['csr', 'csc', 'coo'],
-                        dtype=np.float64)
+                        dtype='numeric')
```

Comment:
Are you sure the test currently fail when you stick with `np.float64`?
It seems to me that it only test `slef.coef_`. Not sure how to test it though...